bitnami/matomo/templates/_helpers.tpl

```diff
@@ -142,3 +142,76 @@ mariadb-password
 db-password
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the matomo image needed initContainers
+*/}}
+{{- define "matomo.initContainers" -}}
+{{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+- name: volume-permissions
+  image: {{ include "matomo.volumePermissions.image" . }}
+  imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+  command:
+    - sh
+    - -c
+    - |
+      mkdir -p "/bitnami/matomo"
+      chown -R "{{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "/bitnami/matomo"
+  securityContext:
+    runAsUser: 0
+  {{- if .Values.volumePermissions.resources }}
+  resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
```

Comment:
You're using `nindent 12` here and `nindent 8` when using the helper, see in `deployment.yaml`:
```yaml
        {{ include "matomo.initContainers" . | nindent 8 }}
```
Please be careful and ensure the indentations you use here are relative:
```suggestion
  resources: {{- toYaml .Values.volumePermissions.resources | nindent 4 }}
```

Reply:
Good catch! Fixed [d42183d](https://github.com/bitnami/charts/pull/18754/commits/d42183d1ae6f096356f815db7636069a8e8f068f)